internal/run.bzl

```
@@ -148,6 +163,9 @@ _postcss_run = rule(
         "sourcemap": attr.bool(default = False),
         "data": attr.label_list(allow_files = True),
         "named_data": attr.label_keyed_string_dict(allow_files = True),
+        "plugins": attr.label_keyed_string_dict(
+            mandatory = True,
+        ),
```

Style nit, also below

```suggestion
        "plugins": attr.label_keyed_string_dict(mandatory = True),
```